cnv/cnv_users_guide/cnv-using-the-default-pod-network-with-cnv.adoc

```diff
@@ -0,0 +1,31 @@
+[id="cnv-using-the-default-pod-network-with-cnv"]
+= Using the default Pod network with {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-using-the-default-pod-network-with-cnv
+toc::[]
+
+You can use the default Pod network with {ProductName}. To do so,
+you must use the `masquerade` binding method. It is the only recommended
+binding method for use with the default Pod network. Do not use
+`masquerade` mode with non-default networks.
+
+[NOTE]
+====
+For secondary networks, use the `bridge` binding method.
+====
+
+include::modules/cnv-configuring-masquerade-mode-cli.adoc[leveloffset=+1]
+
+== Web console
+
+If you create a virtual machine from the {ProductName}
+xref:../../cnv/cnv_users_guide/cnv-create-vms.adoc#cnv-create-vms[web console wizard],
```

Comment:
I don't see this file existing yet in the current PR or repo. Is this going to be a later addition?

Reply:
Yup! Sorry for the confusion. It's going in via @bgaydosrh's PR: https://github.com/openshift/openshift-docs/pull/15529

Reply:
Sounds good!